inductiva/tasks/__init__.py

```diff
@@ -1,4 +1,4 @@
 #pylint: disable=missing-module-docstring
 from inductiva.tasks.task import Task
-from inductiva.tasks.run_simulation import run_simulation
+from inductiva.tasks.run_simulation import run_simulation, TASK_METADATA_FILENAME
```

Comment:
Given this change, let's match this `__init__.py` file as the other ones, and just do `from .task.run_simulation`